opa-react-demo/Caddyfile

```diff
@@ -1,5 +1,5 @@
 :4000 {
-    handle_path /eopa/* {
+    handle /v1/* {
```

Comment:
☝️ This makes Caddy *not* drop `/v1/` from the request path. When using `handle_path` (like before), the configured prefix would be stripped. With `handle`, it's passed along as-is.